samples/net/sockets/http_get/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+    description: BSD Sockets API HTTP GET example
+    name: socket_http_get
+tests:
+-   test:
+        build_only: true
+        platform_whitelist: qemu_x86
```

Comment:
Instead of whitelisting only x86 qemu, we should be able to compile this test in all platforms. Or if not possible, at least arm and x86 qemu should be mentioned here.

Reply:
This is copy-paste from other samples. Removing and let's see what CI says.